includes/sanitizers/class-amp-tag-and-attribute-sanitizer.php

```diff
@@ -656,6 +656,15 @@ private function validate_tag_spec_for_node( $node, $tag_spec ) {
 			return false;
 		}
 
+		// @todo Since these are not just validating but sanitizing the descendant tags, should we move these two into a separate method?
+		if ( ! empty( $tag_spec[ AMP_Rule_Spec::CHILD_TAGS ] ) ) {
+			$this->remove_disallowed_children( $node, $tag_spec[ AMP_Rule_Spec::CHILD_TAGS ] );
+		}
+
+		if ( ! empty( $tag_spec[ AMP_Rule_Spec::DESCENDANT_TAG_LIST ] ) ) {
+			$this->remove_disallowed_descendants( $node, $tag_spec[ AMP_Rule_Spec::DESCENDANT_TAG_LIST ] );
```

Comment:
^ @westonruter:
I added these two methods at this moment for handling the descendants, one for handling direct children only and one for all descendants. Let me know if this seems reasonable to you.